embedded-hal-bus/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(feature = "async", feature(async_fn_in_trait, impl_trait_projections))]
+#![allow(stable_features)]
```

Comment:
It might be worth documenting somewhere why we need this and when we can remove it.

Reply:
done